torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -414,6 +415,29 @@ export default function Hud() {
     params.repoOwner != null && params.repoName != null && params.branch != null
       ? ` (${params.repoOwner}/${params.repoName}: ${params.branch})`
       : "";
+
+  // Get data about viable/strict lag for the current repo, if it exists
+  const strictLagParams: RocksetParam[] = [
+    {
+      name: "repo",
+      type: "string",
+      value: params.repoOwner,
+    },
+    {
+      name: "head",
+      type: "string",
+      value: params.repoOwner === "pytorch" ? "refs/heads/master" : "refs/heads/main",
+    },
+  ];
+  const url = `/api/query/metrics/strict_lag_sec?parameters=${encodeURIComponent(
+    JSON.stringify(strictLagParams)
+  )}`;
+  const { data } = useSWR(url, fetcher, {
+    refreshInterval: 5 * 60 * 1000, // refresh every 5 minutes
+  });
+
+  const viableStrictLag = data === undefined || data.length === 0 ? undefined : `viable/strict lag: ${data[0]["strict_lag_sec"]}h`;
```

Comment:
The units are off here:

```suggestion
  const viableStrictLag = data === undefined || data.length === 0 ? undefined : `viable/strict lag: ${data[0]["strict_lag_sec"]} sec`;
```

Though I'd suggest actually changing the units to be the most course possible ("X hours, Y minutes) to make it easier understand.

Our metrics page uses the format "X.Yh", which is also okay